shell/platform/windows/client_wrapper/flutter_engine_unittests.cc

```diff
@@ -20,6 +20,7 @@ class TestFlutterWindowsApi : public testing::StubFlutterWindowsApi {
   FlutterDesktopEngineRef EngineCreate(
       const FlutterDesktopEngineProperties& engine_properties) {
     create_called_ = true;
+    engine_properties_ = engine_properties;
```

Comment:
Quick update: this is crashing on LUCI and I can reproduce it on my end. The tl;dr is that `engine_properties` is deep copied, but that includes just copying the `dart_entrypoints_argv` pointer itself, which is then invalidated when `FlutterEngine`'s ctor returns. Figuring out a good solution now...

Reply:
So I ended up just deep copying the dart_entrypoint_argv into a std::vector<std::string> in the stubbed call to EngineCreate. I considered a few alternatives but the complexity didn't seem worth it for a single test case.

We could also possibly look into pulling in FlutterProjectBundle to the test harness and initialising that from the DartProject (which is what the normal engine codepath does) but strictly speaking FlutterProjectBundle is not part of client_wrapper.